L1Trigger/L1THGCal/interface/backend/HGCalStage2ClusterDistribution.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __L1Trigger_L1THGCal_HGCalStage2ClusterDistribution_h__
+#define __L1Trigger_L1THGCal_HGCalStage2ClusterDistribution_h__
+
+#include <array>
+#include <string>
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
+
+#include "DataFormats/L1THGCal/interface/HGCalTriggerCell.h"
+#include "DataFormats/L1THGCal/interface/HGCalCluster.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerTools.h"
+
+#include "L1Trigger/L1THGCal/interface/HGCalBackendDetId.h"
+
+class HGCalStage2ClusterDistribution {
+
+public:
+  HGCalStage2ClusterDistribution(const edm::ParameterSet& conf);
+
+HGCalTriggerGeometryBase::geom_set getStage2FPGAs(const unsigned stage1_fpga, const HGCalTriggerGeometryBase::geom_set stage2_fpgas, const edm::Ptr<l1t::HGCalCluster>& tc_ptr) const;
```

Comment:
Would be better to use a const reference for `stage2_fpgas` and avoid copy, since it is a set

Reply:
thanks, changed